docs/source/configure.rst

```diff
@@ -86,6 +86,22 @@ To view the status of the server after it is running, enter the following comman
 
   # sudo docker ps
```

Comment:
Do we need to provide a way for them to set this in the install script too? That way they can just run the install script and pass it in using `-e QPC_SERVER_TIMEOUT=30` in addition to being able to manually do it starting the docker container?

Reply:
Done